### ISSUE TITLE
A 1.20.1 version of the previous system 

### ISSUE THREAD

**Issue by porridge-eater** [Globox1997/VillagerQuests#42]
### Make sure you are not opening a duplicate.

- [ ] I have searched the issue tracker and did not find an issue describing my bug.

### Description of your suggestion.

I like the new rework. But would it be possible to have a simpler version, too? Just a version update of the intital release's system. Just a suggestion for those who might like the other system more for their gameplay.

**Comment by Globox1997** [comment]
I could add a villager profession option plus profession tier to a quest in ftbquests.
So you can create quests easily with ftbquests and those would then just get randomly added to villagers with the profession.

What do you think?

**Comment by porridge-eater** [comment]
I think so. I personally don't think I'd use the ftbquests system. I wouldn't mind the old system with data packs. But if this new system is more intuitive, then that's fine. Thank you for considering my suggestion, nonetheless.

**Comment by MJRamon** [comment]
Yeah, the whole FTB ecosystem just for villager quests is too heavy.

**Comment by Toxic62** [comment]
> I could add a villager profession option plus profession tier to a quest in ftbquests. So you can create quests easily with ftbquests and those would then just get randomly added to villagers with the profession.
> 
> What do you think?

Heya, are you still planning to add that ? It could be great since working with uuids only isn't good between worlds as you can't use natural villagers for the quests
